blazar/api/v1/utils.py

```diff
@@ -94,9 +95,12 @@ def handler(**kwargs):
                     except ex.BlazarException as e:
                         return bad_request(e)
                     except messaging.RemoteError as e:
-                        # Get the exception from manager and common exceptions
-                        cls = getattr(manager_exceptions, e.exc_type,
+                        # Get the exception from enforcement, manager and
+                        # common exceptions
+                        cls = getattr(enforcement_exceptions, e.exc_type,
```

Comment:
Think it's my first time seeing this code but I find this kind of weird. It will try to default to the generic `ex` module pretty quickly. I guess it makes sense, it assumes there are no collisions b/w all the exception modules.